modules/search/class.jetpack-search.php

```diff
@@ -199,11 +199,19 @@ public function init_hooks() {
 	 */
 	public function load_assets() {
 		if ( defined( 'JETPACK_SEARCH_PROTOTYPE' ) ) {
-			$script_relative_path = '_inc/search/dist/jp-search.bundle.js';
+			$script_relative_path = '_inc/build/instant-search/jp-search.bundle.js';
 			if ( file_exists( JETPACK__PLUGIN_DIR . $script_relative_path ) ) {
 				$script_version = self::get_asset_version( $script_relative_path );
 				$script_path    = plugins_url( $script_relative_path, JETPACK__PLUGIN_FILE );
-				wp_enqueue_script( 'jetpack-search', $script_path, array(), $script_version, false );
+				wp_enqueue_script( 'jetpack-search', $script_path, array(), $script_version, true );
+				wp_add_inline_script( 'jetpack-search', 'window.JetpackInstantSearchOptions = { siteId: ' . Jetpack::get_option( 'id' ) .'}', 'before' );
+			}
+
+			$style_relative_path = '_inc/build/instant-search/instant-search.min.css';
```

Comment:
I think it would be nice to enqueue an unminified version of the CSS when `SCRIPT_DEBUG` is set to true. We usually do that in Jetpack, to help with debugging:
https://github.com/Automattic/jetpack/blob/b96f66a73398cd79dd460127fa07855ee4b85fcf/_inc/social-logos.php#L10

Reply:
👍 Added this to the master issue (#13027).